plugins/database/src/db_plugin.cpp

```diff
@@ -2229,6 +2231,18 @@ irods::error db_mod_data_obj_meta_op(
                     }
                 }
             }
+            if(regParamNames[i] == DATA_ACCESS_TIME_KW) {
+                /* if access_ts, also make sure it's in the standard time-stamp format: "%011d" */
+                if (colNames[i] == "access_ts") { /* double check*/
+                    if ( strlen( theVal ) < 11 ) {
+                        static char theVal3[20];
```

Comment:
i don't think that it matters... but the earlier blocks incremented...

```suggestion
                        static char theVal4[20];
```

and of course, the next 3-4 lines would do the same...

Reply:
done